packages/babel-jest/src/__tests__/index.ts

```diff
@@ -93,7 +93,14 @@ describe('caller option correctly merges from defaults and options', () => {
 
     expect(loadPartialConfig).toHaveBeenCalledTimes(1);
     expect(loadPartialConfig).toHaveBeenCalledWith(
-      expect.objectContaining({caller: {name: 'babel-jest', ...output}}),
+      expect.objectContaining({
```

Comment:
good example of the use case of `expect.objectContaining` current behavior - I only care about a subset of the properties, but I care about all of _their_ properties


should probably add a test

Reply:
added here: [`cda33c9` (#10766)](https://github.com/facebook/jest/pull/10766/commits/cda33c9c71cccf6790922f2ac366d466bede4840)